node/bft/src/worker.rs

```diff
@@ -318,18 +318,20 @@ impl<N: Network> Worker<N> {
         // Construct the transmission.
         let transmission = Transmission::Solution(solution.clone());
         // Compute the checksum.
-        let checksum = solution.to_checksum();
+        let checksum = solution.to_checksum::<N>()?;
```

Comment:
I know I added this before, but for the same reason as my comment above, I think we should discard these checksum calculations and just stick to tracking by {solution, transaction}_id.

Reply:
That would require overhauling the `pending` and  `ready` objects and the underlying `fn contains_transmission`.

Reply:
If you think that's too much for this PR, we can leave this and the above comment for future work.

Reply:
Ideally, we try to minimize changes, especially since the introduction of checksums seems to already touch so many surfaces.